### PR TITLE
SPUI 719 > News screen

### PR DESCRIPTION
**Pull request by DenisMartych**

### CODE REVIEW

File: lib/service/services/education_news/model/education_news_response_model.dart [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    required List<String> associatedAssets,
+    required String description,
+  }) = _EducationNewsModel;
+
+  factory EducationNewsModel.fromJson(Map<String, dynamic> json) =>
+      _$EducationNewsModelFromJson(json);
+}
+
+enum Sentiment {
+@JsonValue('Neutral')
```
Comment (eli1stark) [inline lib/service/services/education_news/model/education_news_response_model.dart:34]: add spaces

File: plugins/simple_kit/lib/src/news/components/simple_smile_icon.dart [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const SSmileIcon({
+    Key? key,
+    required this.sentiment,
+  }) : super(key: key);
+
+  final Color sentiment;
+
+  @override
+  Widget build(BuildContext context) {
+    if (sentiment == Colors.green) {
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/news/components/simple_smile_icon.dart:14]: use SColorsLight()

File: plugins/simple_kit/lib/src/news/components/simple_smile_icon.dart [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      return const SSmileGoodIcon();
+    }
+    if (sentiment == Colors.yellow) {
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/news/components/simple_smile_icon.dart:17]: use SColorsLight()

File: plugins/simple_kit/lib/src/news/simple_news_category.dart [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  final String timestamp;
+  final Function() onTap;
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
+      onTap: onTap,
+      child: SPaddingH24(
+        child: Container(
+          padding: EdgeInsets.symmetric(vertical: 20.h),
```
Comment (eli1stark) [inline plugins/simple_kit/lib/src/news/simple_news_category.dart:29]: comma

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/screens/education/education.dart [hunk 5]
```diff
@@ -1,58 +1,78 @@
...
-                    builder: (context) {
-                      return const ExampleScreen();
-                    },
-                  ),
-                );
-              },
-              child: const Text(
-                'Example Screen',
-              ),
+    _scrollController.addListener(() {
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/app/screens/education/education.dart]: done

File: lib/app/screens/education/education.dart [hunk 6]
```diff
@@ -1,58 +1,78 @@
...
+      },
+      loading: () => const Loader(),
+      error: (_, __) => Container(),
     );
   }
+
+  Color _newsColor(Sentiment sentiment) {
+    switch (sentiment) {
+      case Sentiment.neutral:
+        return Colors.yellow;
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/app/screens/education/education.dart]: done

File: lib/app/screens/education/education.dart [hunk 7]
```diff
@@ -1,58 +1,78 @@
...
+      case Sentiment.positive:
+        return Colors.green;
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/app/screens/education/education.dart]: done

File: lib/app/screens/education/education.dart [hunk 8]
```diff
@@ -1,58 +1,78 @@
...
+      case Sentiment.negative:
+        return Colors.red;
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/app/screens/education/education.dart]: done

File: lib/app/shared/features/education_news/provider/education_news_fpod.dart [hunk 9]
```diff
@@ -0,0 +1,28 @@
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+import '../../../../../service/services/education_news/model/education_news_request_model.dart';
+import '../../../../../service/services/education_news/model/education_news_response_model.dart';
+import '../../../../../shared/providers/service_providers.dart';
+import '../notifier/education_news_notipod.dart';
+
+final educationNewsInitFpod =
+FutureProvider.autoDispose<List<EducationNewsModel>>((ref) async {
+  final educationNewsService = ref.read(educationNewsServicePod);
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/app/shared/features/education_news/provider/education_news_fpod.dart]: done

File: lib/app/shared/features/support/view/support.dart [hunk 10]
```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:simple_kit/example/example_screen.dart';
+import 'package:simple_kit/simple_kit.dart';
+
+class Support extends HookWidget {
```
[the remark this answers is not in the document]
Comment (DenisMartych) [reply lib/app/shared/features/support/view/support.dart:7]: done

File: lib/service/services/education_news/model/education_news_response_model.dart (hunk 1 above)
Comment (DenisMartych) [reply lib/service/services/education_news/model/education_news_response_model.dart]: done

File: plugins/simple_kit/lib/src/news/components/simple_smile_icon.dart (hunk 2 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/news/components/simple_smile_icon.dart:14]: done

File: plugins/simple_kit/lib/src/news/components/simple_smile_icon.dart (hunk 3 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/news/components/simple_smile_icon.dart:17]: done

File: plugins/simple_kit/lib/src/news/simple_news_category.dart (hunk 4 above)
Comment (DenisMartych) [reply plugins/simple_kit/lib/src/news/simple_news_category.dart:29]: done

**Pull request merged**